01_Inleiding.Rmd

```diff
@@ -65,7 +67,8 @@ kable(gw_types_groupen,
 ```
     
 
-_GT-groep 5 bevat uitsluitend grondwaterafhankelijke HT die zodanig breed gedefinieerd zijn dat ze op sommige locaties grondwatergevoed zijn, maar soms tot meestalniet._ **_De vraag is of we deze groep al dan niet meenemen._**  
+_GT-groep 5 bundelt grondwaterafhankelijke HT die zodanig breed gedefinieerd zijn dat ze op sommige locaties grondwatergevoed zijn, maar soms tot meestal niet.
+_ **_De vraag is of we deze groep meenemen._**  
 De GT-groepen lenen zich heel goed om een stratificatie uit te voeren op basis van de verwachte responssnelheid van een locatie op droogte. 
 De permanent gevoede locaties (GT-groep 1) zullen naar verwachting trager reageren dan de tijdelijk of zwak gevoede locaties. 
 Door een stratificatie over de GT-groepen toe te passen kan een _zekere_ **_(nog af te spreken)_** balans worden verzekerd. 
```

Comment:
hier kun je ook de overweging meegeven of we per groep uitspraken willen doen (lijkt me zinvol), dan wel (stratificatie s.s.) alleen overall

Reply:
OK, dit als discussiepunt toegevoegd.